projects/app-lob/src/app/grid-finjs/grid-finjs.component.html

```diff
@@ -3,23 +3,24 @@
         igxPreventDocumentScroll
         width="100%" height="30%"
         displayDensity='compact'
-        columnHidingTitle="Indicators"
         hiddenColumnsText="Hidden"
         primaryKey='ID'
-        exportExcelText="Export to Excel"

```

Comment:
The ability to set excel export text and csv export text should be exposed as well. Example:

```
<igx-grid-toolbar-exporter [exportCSV]="false" [exportExcel]="true">
        <span excelText>Transform to Excel</span>
        <span csvText>Transform to CSV</span>
</igx-grid-toolbar-exporter>
```

I assume the default values of exportCSV and exportExcel equivalents was 'false' and thats the reason to explicitly set it?

Please do consider this for all following examples below.

Reply:
Also, we should check why there is no equivalent for "hiddenColumnsText" which will be deprecated. We might have be facing a bug here ;)